docs/source/tutorials/create-blastdb-metadata.rst

```diff
@@ -32,6 +32,26 @@ buckets, but if you are working with your own BLAST database, you will benefit
 from creating the metadata file and uploading it to the cloud alongside the
 BLAST database files. This tutorial will show you how to do that.
 
+**Note:**
+If you create a BLAST database using ``makeblastdb`` version 2.13 or greater,
+you do not need to use the ``create-blastdb-metadata.py`` script. Instead,
+you can use the ``metadata_output_prefix`` command line option to specify
```

Comment:
The `metadata_output_prefix` is not really needed, because elastic-blast ignores the `files` field in metadata. I am not sure if we need to complicate users' lives with this.

Reply:
Thanks for the feedback, I simplified the instructions in [d5bfbb6](https://github.com/ncbi/elastic-blast-docs/pull/144/commits/d5bfbb6f298273ead7e073575ef76bb6de3295cd)

Reply:
Thanks for the changes!